### PR TITLE
Initial prices API documentation

### LINKED ISSUE #46: Add documentation for `v1/prices`

### PR DESCRIPTION
**Pull request by Jeadie**

### CODE REVIEW

File: reference/sql-query-tables/prices/README.md [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+
+The columns and their schema available for each table can be viewed with the `describe <table>` command. For example:
+
+```sql
+/* Show the columns available */
+DESCRIBE spiceai.prices;
+DESCRIBE spiceai.assets;
+```
+
+### Per-pair price base type tables available to query
```
Comment (Jeadie) [inline reference/sql-query-tables/prices/README.md:23]: We should decide whether its worth continuing to publicise these `prices.[Token0]_[Token2]`  datasets.

File: reference/sql-query-tables/prices/README.md [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
 # Prices
 
+### Aggregate price base type tables available to query
+
+| View Name                                          | Description                                      |
+| -------------------------------------------------- | ------------------------------------------------ |
+| [`prices.trades`](trades.md)                       | Raw trade data from exchanges                    |
+| [`prices.trades_detailed`](trades\_detailed.md)    | Trade data with additional columns               |
+| [`prices.assets`](assets.md)                       | Tokens supported on Spice AI with metadata       |
+| [`prices.minute_resolution`](prices.md)                      | Per-minute price data for tokens pairs           |
```
Comment (Jeadie) [inline reference/sql-query-tables/prices/README.md]: Need to confirm with team on naming `prices.minute_resolution`. I also thought `prices.ohlc_minute`

File: reference/sql-query-tables/prices/assets.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# Asset data
+
+All assets verified and useable within SpiceAI. 
+
+| Column Name     | Data Type         | Description                                                         |
+| --------------- | ----------------- | ------------------------------------------------------------------- |
+| asset_id        | CHARACTER VARYING | Unique identifier for the asset across SpiceAI platform             |
+| name            | CHARACTER VARYING | Human readable name of the underlying asset/token                   |
```
Comment (phillipleblanc) [inline reference/sql-query-tables/prices/assets.md]:
```suggestion
| name            | CHARACTER VARYING | Display name of the underlying asset/token                   |
```

File: reference/sql-query-tables/prices/README.md [hunk 4]
```diff
@@ -6,18 +6,16 @@ description: Price data tables available in SQL query
...
 
-| View Name                                          | Description                                      |
-| -------------------------------------------------- | ------------------------------------------------ |
-| [`prices.trades`](trades.md)                       | Raw trade data from exchanges                    |
-| [`prices.trades_detailed`](trades\_detailed.md)    | Trade data with additional columns               |
-| [`prices.assets`](assets.md)                       | Tokens supported on Spice AI with metadata       |
-| [`prices.minute_resolution`](prices.md)            | Per-minute price data for tokens pairs           |
+| View Name                          | Description                                      |
+| ---------------------------------- | ------------------------------------------------ |
+| [`prices.all_pairs`](prices.md)    | Per-minute price data for tokens pairs           |
```
Comment (phillipleblanc) [inline reference/sql-query-tables/prices/README.md]:
```suggestion
| [`prices.all_pairs`](prices.md)    | Per-minute price data for supported token pairs           |
```

File: reference/sql-query-tables/prices/prices.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ---
- description: SQL table schema for prices.minute_resolution
+ description: SQL table schema for prices.all_pairs
 ---
  
 # Price data
 
-Aggregated price data for all tokens, bucketed per minute.
+Aggregated OHLC price data for all tokens, per minute resolution.
```
Comment (phillipleblanc) [inline reference/sql-query-tables/prices/prices.md]:
```suggestion
Aggregated OHLC price data for all supported tokens, per minute resolution.
```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**